packages/unit-testing/stubs/commerce/webstore/ShopperInstance.js

```diff
@@ -0,0 +1,90 @@
+define([], function () {
```

Comment:
From documentation I can't find the following modules:
   commerce/webstore/ShopperInstance
   commerce/webstore/order
   commerce/webstore/shopper
   commerce/promising

Reply:
That's true, not sure why they are not documented even though they are in the API files, I would keep them here, should give additional stubs to commerce developers